content/webapp/pages/search/events.tsx

```diff
@@ -242,10 +242,8 @@ export const getServerSideProps: GetServerSideProps<
     };
   }
 
-  // Sending page=1 to Prismic skips the two first results, which seems to have to do with the cursor work
-  // This is a workaround that ensures we only send the page if relevant
```

Comment:
Leftover from when we were using Prismic in search... I tested and I do think it could be removed.